src/compiler/checker.ts

```diff
@@ -1776,6 +1777,20 @@ namespace ts {
             }
         }
 
+        function tryGetMemberInModuleExportsAndProperties(memberName: __String, moduleSymbol: Symbol): Symbol | undefined {
+            const symbolTable = getExportsOfModule(moduleSymbol);
+            if (symbolTable) {
+                let symbol = symbolTable.get(memberName);
+                if (!symbol) {
+                    const exportEquals = resolveExternalModuleSymbol(moduleSymbol);

```

Comment:
The export properties don't obviously depend on `symbolTable`.  Is there a reason they're not checked if `symbolTable` is `undefined`?  It looks like `getExportsAndPropertiesOfModule`, above, doesn't have this restriction, but it may simply be missing an optimization.